code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2343,10 +2418,34 @@
 						to_chat(H, "<b><span class='warning'>Something bursts out from inside you!</span></b>")
 						message_admins("[key_name(H)] has gibbed and spawned a new cyber horror due to nanobots. ([formatJumpTo(H)])")
 						H.visible_message("<b><span class='warning'>[H]'s body rips aparts to reveal something underneath!</b></span>")
-						new /mob/living/simple_animal/hostile/monster/cyber_horror(H.loc)
-						spawning_horror = 0
-						H.gib()
-
+						if(H.species.name == "Vox")
```

Comment:
Put a variable on species that gives the cyber horror type and then use that variable here instead of this